deptofdefense/move.mil
# Add High Value Inventory Worksheet page content

## Checklist

I have…


## Summary of Changes

This pull request adds content to the High Value Inventory Worksheet page, including a downloadable PDF version of the worksheet.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-high-value-inventory-worksheet`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails server`,
1. load up http://localhost:3000/resources/inventory-form

## Screenshots

<details>
  <summary>Narrow Viewport</summary>

  ![inventory-worksheet-narrow](https://user-images.githubusercontent.com/27780860/38944430-d97dece2-4301-11e8-8baf-dad19cb87ccd.png)
</details>

<details>
  <summary>Wide Viewport</summary>

  ![inventory-worksheet-wide](https://user-images.githubusercontent.com/27780860/38944433-dd1d9352-4301-11e8-9d20-859047111fc4.png)
</details>
